static/app/components/devtoolbar/components/releases/releaseIssues.tsx

```diff
@@ -0,0 +1,142 @@
+import InfiniteListState from 'sentry/components/devtoolbar/components/infiniteListState';
+import PanelLayout from 'sentry/components/devtoolbar/components/panelLayout';
+import useInfiniteReleaseIssuesList from 'sentry/components/devtoolbar/components/releases/useInfiniteReleaseIssuesList';
+import SentryAppLink from 'sentry/components/devtoolbar/components/sentryAppLink';
+import useConfiguration from 'sentry/components/devtoolbar/hooks/useConfiguration';
+import {
+  badgeWithLabelCss,
+  gridFlexEndCss,
+  listItemGridCss,
+  listItemPlaceholderWrapperCss,
+} from 'sentry/components/devtoolbar/styles/listItem';
+import {
+  panelInsetContentCss,
+  panelSectionCss,
+} from 'sentry/components/devtoolbar/styles/panel';
+import {infoHeaderCss} from 'sentry/components/devtoolbar/styles/releasesPanel';
+import {resetFlexColumnCss} from 'sentry/components/devtoolbar/styles/reset';
+import {smallCss, xSmallCss} from 'sentry/components/devtoolbar/styles/typography';
+import TimesTag from 'sentry/components/group/inboxBadges/timesTag';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import PanelItem from 'sentry/components/panels/panelItem';
+import Placeholder from 'sentry/components/placeholder';
+import TextOverflow from 'sentry/components/textOverflow';
+import TimeSince from 'sentry/components/timeSince';
+import type {Group} from 'sentry/types/group';
+
+const estimateSize = 89;
+const placeholderHeight = `${estimateSize - 8}px`; // The real height of the items, minus the padding-block value
+
+export default function ReleaseIsssues({releaseVersion}: {releaseVersion: string}) {
+  const queryResult = useInfiniteReleaseIssuesList({
+    releaseVersion,
+  });
+
+  return (
+    <PanelLayout>
+      <PanelItem css={{padding: 'var(--space150)'}}>
+        <div css={infoHeaderCss}>Latest issues related to this release</div>
+      </PanelItem>
+      <div css={resetFlexColumnCss}>
+        <InfiniteListState
+          queryResult={queryResult}
+          backgroundUpdatingMessage={() => null}
+          loadingMessage={() => (
+            <div
+              css={[
+                resetFlexColumnCss,
+                panelSectionCss,
+                panelInsetContentCss,
+                listItemPlaceholderWrapperCss,
+              ]}
+            >
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+            </div>
+          )}
+        >
+          <InfiniteListItems
+            estimateSize={() => estimateSize}
+            queryResult={queryResult}
+            itemRenderer={props => <IssueListItem {...props} />}
+            emptyMessage={() => <p css={{panelInsetContentCss}}>No items to show</p>}
+          />
+        </InfiniteListState>
+      </div>
+    </PanelLayout>
+  );
+}
+
+function IssueListItem({item}: {item: Group}) {
```

Comment:
this whole component is almost copied exactly from the one in `issuesPanel.tsx` except for the analytics -- should we rewrite the component to be reusable??

Reply:
yeah, if you want to play with it and do something then get in there!

Reply:
or like, put `IssueListItem` into it's own file and import it in both places.

Reply:
for analytics i was thinking of using some react Providers to pass down context, and build up the parts of `devtoolbar.release-issue-list.item.click` with nesting. I can show you what i was thinking if you want to refactor that first across the toolbar. There'll be loads of copy+paste revealed afterwards